deployment/v3/utils/prop_migrator/knowledge/old-value-takes-priority.csv

```diff
@@ -49,7 +49,6 @@ registration-processor-default.properties,registration.processor.reprocess.hours
 registration-processor-default.properties,mosip.registration.processor.database.port
 registration-processor-default.properties,registration.processor.LANDING_ZONE
 registration-processor-default.properties,mosip.regproc.packet.classifier.tagging.agegroup.ranges
-registration-processor-default.properties,mosip.kernel.virus-scanner.host
```

Comment:
This property is removed from upgrade2 and 1.2.0.1-V3 config branch because the same property key is present in "application-default.properties" so we do not need this property in "registration-processor-default.properties"